updateConfig.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import sys, re, os, json, gzip
+from argparse import ArgumentParser
+from os import path, environ
+from subprocess import  call, check_output
+
+base = sys.argv[0]
+help_text = "\n\
+    %s --project <name> --version <version> --scram <scram version>\n\
+    --toolbox <toolbox> [--config <dir>] [--arch <arch>] [--help]\n\n\
+    This script will copy all <name>_<files> files into <files>\n\
+    and replace project names, version, scram verion, toolbox path\n\
+    and extra keys/values provided via the command line. e.g.\n\n\
+    %s -p CMSSW -v CMSSW_4_5_6 -s V1_2_0 -t /path/cmssw-tool-conf/CMS170\
+ --keys MYSTRING1=MYVALUE1 --keys MYSTRING2=MYVALUE2\n\n\
+    will release:\n\
+    @PROJECT_NAME@=CMSSW\n\
+    @PROJECT_VERSION@=CMSSW_4_5_6\n\
+    @SCRAM_VERSION@=V1_2_0\n\
+    @PROJECT_TOOL_CONF@=/path/cmssw-tool-conf/CMS170\n\
+    @MYSTRING1@=MYVALUE1\n\
+    @MYSTRING2@=MYVALUE2\n\n" % (base, base)
+parser = ArgumentParser(usage=help_text)
+parser.add_argument('--project', '-p', dest='project', required=True, help='Missing or empty project name.')
+parser.add_argument('--version', '-v', dest='version', required=True, help='Missing or empty project version.')
+parser.add_argument('--scram', '-s', dest='scram', required=True, help="Missing or empty scram version.")
+parser.add_argument('--toolbox', '-t', dest='toolbox', required=True, help="Missing or empty SCRAM tool box path.")
+parser.add_argument('--config', dest='config')
+parser.add_argument('--keys', dest='keys', action='append')
+parser.add_argument('--arch', '-a', dest='arch')
+args = parser.parse_args()
+if len(sys.argv) < 3:
+    parser.print_usage()
+    sys.exit(1)
+project = args.project
+version = args.version
+scram = args.scram
+toolbox = args.toolbox
+config = args.config
+keys = dict(s.split('=') for s in args.keys)
+arch = args.arch
+
+
+def fixPath(dir):
```

Comment:
@tulamor , there are build python fucntions to do this . I think os.path.abspath(dir) shoudl return the same as fixPath(dir). please test and use abspath